airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -335,3 +337,37 @@ def get_task_instances(
         ],
         total_entries=total_entries,
     )
+
+
+@task_instances_router.get(
+    "/{task_id}/tries/{task_try_number}", responses=create_openapi_http_exception_doc([401, 403, 404])
```

Comment:
401 and 403 are now added to the base router. Refer to https://github.com/apache/airflow/pull/43932
```suggestion
    "/{task_id}/tries/{task_try_number}", responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND])
```